packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.spec.ts

```diff
@@ -58,35 +58,39 @@ const cardTemplate: string = `<div class="card" style="width: ${cardWidth}px; he
 const getCards = (cnt: number): string => new Array(cnt).fill(cardTemplate).reduce((s, c) => s += c, '');
 
 async function setup() {
-    const { element, connect, disconnect } = await fixture(FASTHorizontalScroll());
+    const { element, connect, disconnect }:
```

Comment:
What was the reason for the explicit typing in this particular case?

Reply:
I found that I had to do a lot of `(element as HorizontalScroll)` to access the class methods and members. Typing it up front in the setup means that I don't have to do that.

Reply:
Hmm. I thought we had the types worked out so that `FASTHorizontalScroll()` would return a typed element from the `fixture` function. Was it typing it as `HorizontalScroll` and that wasn't specific enough?

Reply:
It's probably just an issue with intellisense. It's showing errors everywhere but it builds without errors and tests pass. I just hate the red squiggles.

Reply:
VS Code sometimes caches errors and warnings and I've found restarting it usually reconciles/fixes this. Not to suggest this is your problem, just a tip to resolve that hopefully.  Oddly, I don't see anything in this PR that could break the pipeline.  It seems fairly straightforward.  @robarbms what's you're setup (os, os version, node version, npm version)?